WcaOnRails/config/locales/de.yml

```diff
@@ -2905,6 +2920,10 @@ de:
         zero: Teilnehmer dürfen keine Gäste mitbringen.
         one: Teilnehmer dürfen maximal einen Gast mitbringen.
         other: 'Teilnehmer dürfen maximal %{count} Gäste mitbringen.'
+      #original_hash: bad09c8
+      event_limit:
+        one: Teilnehmer dürfen sich in genau einem Event anmelden.
```

Comment:
Similar case as above https://github.com/thewca/worldcubeassociation.org/pull/7612/files#r1100258867
Now in a view that registrants would see, same treatment by removing the `zero:` entry.